libafl_qemu/src/drcov.rs

```diff
@@ -0,0 +1,145 @@
+use std::{collections::HashSet, path::PathBuf, sync::Mutex};
+
+use hashbrown::{hash_map::Entry, HashMap};
+use lazy_static::lazy_static;
+use libafl::{inputs::UsesInput, state::HasMetadata};
+use libafl_targets::drcov::{DrCovBasicBlock, DrCovWriter};
+use rangemap::RangeMap;
+
+use crate::{
+    blocks::pc2basicblock,
+    emu::GuestAddr,
+    helper::{QemuHelper, QemuHelperTuple, QemuInstrumentationFilter},
+    hooks::QemuHooks,
+    Emulator,
+};
+
+lazy_static! {
+    static ref DRCOV_IDS: Mutex<HashSet<u64>> = Mutex::new(HashSet::new());
```

Comment:
No need for lazy_static if you do it like this:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=d05a324ed6a1bea35c59b2d4faa7d2e2

Reply:
Or use hooks with closures